---
title: Операнд оператора Throw должен быть производным от System.Exception
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: 71727cc191f8ab1663589e64fb9ee5ce076684f4
ms.sourcegitcommit: 2701302a99cafbe0d86d53d540eb0fa7e9b46b36
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 04/28/2019
ms.locfileid: "64646416"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>Операнд оператора Throw должен быть производным от System.Exception
Аргумент для `Throw` должен быть либо экземпляром `System.Exception` , либо экземпляром класса, производного от `System.Exception`.  
  
 **Идентификатор ошибки:** BC30665  
  
## <a name="to-correct-this-error"></a>Исправление ошибки  
  
- Используйте аргумент, производный от `System.Exception`, как показано в приведенном ниже примере.  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>См. также

- [Оператор Throw](../../visual-basic/language-reference/statements/throw-statement.md)
- [Оператор Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [Обработка и создание исключений в .NET](../../standard/exceptions/index.md)
